index.html

```diff
@@ -0,0 +1,216 @@
+              </div>
+            </div>
+          </div>
+          <div class="col-12 col-lg-3 offset-lg-1 weather-details">
+            <ul class="nav">
+              <li>
+                <picture>
+                  <source
+                    srcset="images/icon-feels-like-dark-mode.svg"
+                    media="(prefers-color-scheme: dark)"
+                  />
+                  <img
+                    src="images/icon-feels-like.svg"
+                    alt="temperature feels like"
+                  />
+                </picture>
+                Feels like: <span id="weather-feels"></span>
+              </li>
+              <li>
+                <picture>
+                  <source
+                    srcset="images/icon-wind-dark-mode.svg"
+                    media="(prefers-color-scheme: dark)"
+                  />
+                  <img src="images/icon-wind-speed.svg" alt="wind speed" />
+                </picture>
+                Wind:
+                <span id="wind"></span>
+              </li>
+              <li>
+                <picture>
+                  <source
+                    srcset="images/icon-humidity-dark-mode.svg"
+                    media="(prefers-color-scheme: dark)"
+                  />
+                  <img src="images/icon-humidity.svg" alt="humidity" />
+                </picture>
+                Humidity:
+                <span id="humidity"></span>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <div class="container">
+        <div class="forecast">
+          <h3 class="forecast-title">Your Forecast</h3>
+          <div class="row justify-content-center">
+            <div class="col-12 col-lg-6">
+              <div class="row" id="weather-forecast">
+                <div
+                  class="col-12 col-md-6 d-none weather-forecast-item"
+                  id="weather-forecast-template"
+                >
+                  <div class="forecast-weather-information">
+                    <div class="forecast-icon-section">
+                      <img
+                        src="images/icons/clear-sky-day.svg"
+                        alt=""
+                        class="forecast-icon"
+                      />
+                    </div>
+                    <div class="forecast-weather-summary">
+                      <span class="forecast-day">Monday</span>
+                      <p class="forecast-description">Clear</p>
+                    </div>
+                    <div class="forecast-weather-maximum">
+                      <span class="forecast-max-min">MAX</span>
+                      <p class="forecast-maximum-temperature">31º</p>
+                    </div>
+                    <div class="forecast-weather-minimum">
+                      <span class="forecast-max-min">MIN</span>
+                      <p class="forecast-minimum-temperature">8º</p>
+                    </div>
+                  </div>
```

Comment:
No problem here, but maybe we can keep the data here empty.
```suggestion
                <div
                  class="col-12 col-md-6 d-none weather-forecast-item"
                  id="weather-forecast-template"
                >
                  <div class="forecast-weather-information">
                    <div class="forecast-icon-section">
                      <img class="forecast-icon" />
                    </div>
                    <div class="forecast-weather-summary">
                      <span class="forecast-day">Monday</span>
                      <p class="forecast-description"></p>
                    </div>
                    <div class="forecast-weather-maximum">
                      <span class="forecast-max-min">MAX</span>
                      <p class="forecast-maximum-temperature"></p>
                    </div>
                    <div class="forecast-weather-minimum">
                      <span class="forecast-max-min">MIN</span>
                      <p class="forecast-minimum-temperature"></p>
                    </div>
                  </div>
```